colossalai/nn/optimizer/cpu_adamw.py

```diff
@@ -0,0 +1,124 @@
+import torch
+import math
+
+
+class CPUAdamW(torch.optim.Optimizer):
```

Comment:
why not derive from CPUAdam?
Or CPUAdam and CPUAdamW both derive from CPUAdamBase...
A lot of duplicated code between CPUAdam and CPUAdamW

Reply:
Thanks for the advice, I'll consider it.